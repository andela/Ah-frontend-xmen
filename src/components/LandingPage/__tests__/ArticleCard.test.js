import React from 'react';
import { shallow } from 'enzyme';
import ArticleCardComp from '../ArticleCard';

describe('ArticleCard', () => {
  const props = {
    author: {
      username: 'douglas',
    },
    title: 'New Article',
    read_time: '1 min read',
    description: 'this description',
  };
  const wrapper = shallow(<ArticleCardComp {...props} />);

  it('Should render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
