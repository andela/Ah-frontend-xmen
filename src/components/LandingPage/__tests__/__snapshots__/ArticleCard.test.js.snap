// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleCard Should render without crashing 1`] = `ShallowWrapper {}`;
